SampleGetApi/Tests/BaseClass.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Xml.Linq;
+using AventStack.ExtentReports;
+using SampleGetApi.ReportsHelper;
+using Xunit.Abstractions;
+
+namespace SampleGetApi.Tests;
+
+public class BaseClass : IDisposable
+{
+    protected readonly ITestOutputHelper _output;
+    protected readonly ExtentTest? _test;
+    protected readonly string _currentTestName;
+    private static readonly Dictionary<string, TestResult> _testResults = new Dictionary<string, TestResult>();
```

Comment:
We should be using instance as much as possible. Static isn’t good for all these scenarios

Reply:
Static so that reference can be used more than one TestClass file. We can work around this and make it non static. Will look into it